cr.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Netguru iOS code review task
+//
+
+class paymentViewController: UIViewController {
+    var delegate: PaymentViewControllerDelegate
+    let customView = PaymentView()
+    let payment: Payment?
+
+    func callbacks() {
+        view.backgroundColor = UIColor(displayP3Red: 1, green: 1, blue: 1, alpha: 1)
```

Comment:
good idea with moving callback logic into it's own func, tho setting the view `backgroundColor` is not really the part of callback itself. We can move it to `viewDidLoad` or in our case to `loadView`